# Generated by Django 2.0 on 2018-06-03 23:26

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('his06', '0039_auto_20180604_0204'),
    ]

    operations = [
        migrations.CreateModel(
            name='Sick_his',
            fields=[
                ('sick_id', models.IntegerField(db_column='id', max_length=255, primary_key=True, serialize=False)),
                ('care_id', models.CharField(db_column='care_id', max_length=255, verbose_name='诊疗卡号')),
                ('sick_symtop', models.CharField(db_column='sick_symtop', max_length=255, verbose_name='症状描述')),
                ('result', models.CharField(db_column='result', max_length=255, verbose_name='诊断结果')),
                ('medi_use', models.CharField(db_column='medi_use', max_length=255, verbose_name='处方')),
                ('sick_pro', models.CharField(db_column='sick_pro', max_length=255, verbose_name='检查项目')),
            ],
        ),
        migrations.AlterField(
            model_name='patient',
            name='care_id',
            field=models.CharField(db_column='care_id_num', default='20180604508732', error_messages={0: '该诊疗卡号不存在'}, max_length=255, primary_key=True, serialize=False, verbose_name='诊疗卡号'),
        ),
    ]
